src/HPRT/toolkit/views.py

```diff
@@ -133,13 +133,37 @@ class ScreeningsListView(LoginRequiredMixin, ListView):
     
     def get_context_data(self, **kwargs):
         kwargs = super(ScreeningsListView, self).get_context_data(**kwargs)
+        DSMV_records = DSMV.objects.filter(patient = Patient.objects.filter(pk = self.kwargs['pk'])[0])
```

Comment:
Update to ^ comment - I see that that should still work the same, but I think it would be cleaner together in the kwargs.update function.

Reply:
Each of the queries is called thrice - once for the list of records to display in the table, and twice to get each the x and y axes of the graph. I don't think it would be cleaner to have redundant calls in the kwargs.update function.